pkg/controller/controller.go

```diff
@@ -200,8 +200,14 @@ func sendEvent(obj interface{}, kind, eventType string, err error) {
 	}
 
 	// Send notification to communication chennel
-	notifier := notify.NewSlack()
-	notifier.SendEvent(event)
+	notifier, status := notify.NewSlack()
```

Comment:
no checks if slack/mattermost enabled? Also, can we have some mechanism where for notifying on slack we are not waiting for mattermost and vice versa. Basically notifying can be asynchronous

Reply:
The status flag is added to check if Slack/Mattermost is enabled. Will figure out some mechanism to make asynchronous calls for notifier.

Reply:
@mugdha-adhav let's call SendEvent and SendMessage as a go routine.
ref: https://github.com/infracloudio/botkube/pull/59/files#diff-3c862eb54a8e0e161b534e0c67e5379eR163-R213

Reply:
sure @PrasadG193 changes made respectively and will update in next commit.